src/content/Partnerships.ts

```diff
@@ -111,6 +112,12 @@ export const PartnershipArray: IPartner[] = [
 		description: 'page.partnerships.gatewayfm',
 		link: 'https://gateway.fm/',
 	},
+	{
+		icon: GloDollar,
+		title: 'Glo Dollar',
+		description: 'page.partnerships.glodollar',
+		link: 'https://discord.gg/PUrBVdzzN5',
```

Comment:
should change this to https://www.glodollar.org/

Reply:
@divine-comedian, changed thx